tests/unit/assessment/conftest.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from databricks.sdk.service.compute import DbfsStorageInfo, InitScriptInfo
+from databricks.sdk.service.pipelines import PipelineCluster
+
+
+@pytest.fixture(scope="function")
+def mock_pipeline_cluster():
+    yield [
```

Comment:
Modify workspace_client_mock to store json in a file for pipelines

Reply:
modified the file accordingly.